Layers/ExternalLayer.gd

```diff
@@ -0,0 +1,45 @@
+extends Resource
+
+
+var external_types = ["shp", "json", "tif", "wmts"]
+
+const path_prefix := "res://Geodata"
```

Comment:
I think `user://` would be more appropriate for this in order to work on exported projects